core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -150,6 +151,9 @@ private[spark] class ExecutorAllocationManager(
   // Metric source for ExecutorAllocationManager to expose internal status to MetricsSystem.
   val executorAllocationManagerSource = new ExecutorAllocationManagerSource
 
+  // Flag to measure whether numExecutorTarget could be adjusted
+  val numTargetExecutorAdjustable = new AtomicBoolean(false)
```

Comment:
Can we rename this to `inInitialExecutorsPeriod` and add a comment explaining the criteria for being in this period?